vaadin-text-field-flow-parent/vaadin-text-field-testbench/src/main/java/com/vaadin/flow/component/textfield/testbench/TextFieldElement.java

```diff
@@ -40,6 +40,11 @@ public class TextFieldElement extends TestBenchElement
      * <p>
      * For more complex scenarios that require a full browser simulation of
      * typing, use {@link #sendKeys(CharSequence...)} instead.
+     * <p>
+     * Note, if you are using {@code ValueChangeMode.LAZY} or
+     * {@code ValueChangeMode.TIMEOUT}, value change event is triggered also
+     * without field losing focus, and hence use of
+     * {@link #sendKeys(CharSequence...)} may be better choice in testing.
```

Comment:
I've rebased the PR and now the latest revision of setValue already has one mention of sendKeys here:

https://github.com/vaadin/flow-components/blob/e4008920d352eabf33d82cc9fc9845f97420d2c1/vaadin-text-field-flow-parent/vaadin-text-field-testbench/src/main/java/com/vaadin/flow/component/textfield/testbench/TextFieldElement.java#L40-L42

If you think mentioning value change modes is still helpful, I'd like the comment to explain why using sendKeys is a better option in those modes.